libqtile/window.py

```diff
@@ -1403,9 +1435,9 @@ def _is_in_window(self, x, y, window):
         return (window.edges[0] <= x <= window.edges[2] and
                 window.edges[1] <= y <= window.edges[3])
 
-    def cmd_set_position(self, dx, dy):
+    def cmd_set_position(self, dx, dy, dz=0):
         if self.floating:
-            self.tweak_float(dx, dy)
+            self.tweak_float(dx, dy, dz=dz)
```

Comment:
I think the keyword you want to use here is `z` actually, not `dz`. Also, let's put the keywords on all arguments (this is the only exception in the codebase to what I just said in my previous comment).

Reply:
Because I think the arguments are actually misnamed here. They're passed without keyword, so they end up being `x` and `y` according to `tweak_float` signature.

Reply:
You are absolutely right! Since the name of the method is `cmd_set_position` and not `cmd_change_position`, I would agree that the variables are misnamed.

Reply:
Changed it to `(x, y, z)` (and renamed the corresponding variables, of course).